cli/ballerina-cli/src/main/java/io/ballerina/cli/task/CreateExecutableTask.java

```diff
@@ -95,6 +101,31 @@ public void execute(Project project) {
                 start = System.currentTimeMillis();
             }
             jBallerinaBackend.emit(JBallerinaBackend.OutputType.EXEC, executablePath);
+
+            if (enableNativeImage) {
```

Comment:
We should also check if c2c is enabled here as well. Since the c2c plugin will generate it, we do not have to bother about it. @xlight05 please confirm.

Reply:
Depends, According to the original approach of the spec issue you created, We need to check the --skip-executables flag. We need to finalize the spec issue to be precise :/